Cookbook/Technical-Documents/NewHiresCheckList.md

```diff
@@ -92,6 +92,8 @@ shell script. Here's how to get the iOS project up and running:
 
 ## What's next?
 
+1. Refer to the [Fastlane Match](https://github.com/Babylonpartners/ios-playbook/blob/master/Cookbook/Technical-Documents/FastlaneMatch.md) document for certificates and provisioning profiles related concerns.
```

Comment:
```suggestion
1. Refer to the [Fastlane Match](FastlaneMatch.md#day-to-day-development) document if you want to run the app on your own iOS devices.
```

How about that? 
1. Relative link
1. Takes you directly to the relevant section
1. Clearer purpose of why you'd want to do it

Reply:
Sounds good! 👍